### DISCUSSION TITLE
Ltmeat wasnt found by bypassi?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#685] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
So a while back in a server I was a moderator in, someone made a ticket and said, that they would like to partner. Then they decided to say they had found a goguardian unblocking method. I was curious what method it was, so i asked him(i think he was a boy by the way he talked). He told me about ltmeat and claimed he had found it. Now as most people know.........ltmeat was found by bypassi. But this kid thought he could get away with saying it wasn't bypassi. Now i thought he was trolling, but he may have just been a random kid who saw a video of ltmeat on youtube (i think he found it on youtube since his discord acc was made 5 days before this ticket, i hope it was my video, or it could have been blazer's or tether's). I talked with bypassi, and he didn't seem to really care. But like the kid was trying to make a partnership and he was trying to do that, by saying he was the one who made ltmeat. So another moderator joined and kicked/banned the person.

**Comment by ghost** [comment]
btw i just made it as a poll since it wasnt a bug fix, or a help dissucssion.

**Reply by Aka-but-nice** [reply]
My guy, Bypassi made it, this is not a valid debate. There shouldn't be any controversy going on.

**Reply by ghost** [reply]
Turns it was just an alt.

**Comment by Aka-but-nice** [comment]
> btw i just made it as a poll since it wasnt a bug fix, or a help dissucssion.

If a person said they made Swamp would you believe them? If a person was to say they were the creator of Point Blank would you believe them? No, and no.

**Comment by 3kh0** [comment]
Yes

**Reply by ghost** [reply]
Yes

**Comment by Aka-but-nice** [comment]
```
so i asked him(i think he was a boy by the way he talked). 
``` 
My man assumed he was a man by the way he talked 💀
